packages/plugin-dom/Dom.ts

```diff
@@ -24,32 +25,58 @@ export class Dom<T extends DomConfig = DomConfig> extends JWPlugin<T> {
     };
 
     domMap = new DomMap();
+    editable: HTMLElement;
 
     async start(): Promise<void> {
-        const editable = this.editor._originalEditable;
+        const node = this.configuration.target;
+        if (!node) {
+            this.editable = document.createElement('jw-editable');
```

Comment:
Aren't we missing this ?
```typescript
// Semantic elements are inline by default.	
// We need to guarantee it's a block so it can contain other blocks.	
this.editable.style.display = 'block';
```